src/core/services/order.service.ts

```diff
@@ -49,6 +50,20 @@ const create = async (order: IPurchaseOrder) => {
     })
 }
 
+const purchaseStatusUpdate = async(statusUpdate: IChangePurchaseStatus) => {
+  return await client({
+    data: statusUpdate,
+    method: 'PATCH',
+    url: 'order/updateOrder'
+  })
+    .then( response => {
+      console.log(response.data.message);
+    })
+    .catch( error => {
+      console.log(error)
+    })
```

Comment:
Maybe return something here like a boolean if the updating is success or not and handle it accordingly in the view.

Reply:
done